docs/userGuide/componentsReference/searchbarDoc.md

```diff
@@ -0,0 +1,29 @@
+<frontmatter>
+footer: userGuideFooter.md
+siteNav: userGuideSections.md
+</frontmatter>
+
+<include src="../../common/header.md" />
+
+<div class="website-content">
+  <tip-box border-left-color="#00B0F0">
```

Comment:
You need to add a title, otherwise we won't know what is the name of this component. :P

`# Searchbar`